scripts/scripts.js

```diff
@@ -93,6 +92,7 @@ async function loadEager(doc) {
   if (getMetadata('breadcrumbs').toLowerCase() === 'true') {
     document.body.classList.add('breadcrumbs-enabled');
   }
+  autolinkModals(doc);
```

Comment:
very minor/nitpicky, but I'd probably do this after LCP, and maybe even in loadLazy.

Reply:
Done in https://github.com/adobe/helix-block-collection/pull/37/commits/70a0d5fa0fdf86c06623297ef2542026eb4cb673 👍